packages/web-components/src/kumo/anchor-button/anchor-button.options.ts

```diff
@@ -0,0 +1,83 @@
+import { ButtonAppearance, ButtonShape, ButtonSize } from '../button/button.options.js';
+import type { ValuesOf } from '../../utils/index.js';
+import type { AnchorOptions } from './anchor-button.js';
+
+/**
+ * Anchor Button Appearance constants
+ * @public
+ */
+export const AnchorButtonAppearance = ButtonAppearance;
```

Comment:
do we want to distinguish these names if they are being exported form the ones that will be coming from fluent?

Reply:
I ended up removing this entire file and using the local `kumo/button/button.styles.ts`